frontends/pytorch/test/graph_export/test_script_debug_info.py

```diff
@@ -0,0 +1,26 @@
+# -*- Python -*-
+# This file is licensed under a pytorch-style license
+# See frontends/pytorch/LICENSE for license information.
+
+import torch
+import torch_mlir
+
+# RUN: %PYTHON %s | FileCheck %s
+
+@torch.jit.script
+def add3(t0, t1, t2):
+  intermediate = t0 + t1
+  final = intermediate + t2
+  return final
+
+mb = torch_mlir.ModuleBuilder()
+mb.import_function(add3)
+
+# Verify again with debug info present. Just checking that it makes it in there.
+# CHECK-LABEL: func @add3$generic
+# CHECK: constant 1{{.*}}loc({{.*}}test_script_debug_info.py
```

Comment:
FYI FileCheck does support a thing called `@LINE` which you can use to make this check more precise:
https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-pseudo-numeric-variables